x/auth/vesting/types/codec.go

```diff
@@ -14,3 +16,29 @@ func RegisterCodec(cdc *codec.Codec) {
 	cdc.RegisterConcrete(&DelayedVestingAccount{}, "cosmos-sdk/DelayedVestingAccount", nil)
 	cdc.RegisterConcrete(&PeriodicVestingAccount{}, "cosmos-sdk/PeriodicVestingAccount", nil)
 }
+
+// RegisterInterface associates protoName with AccountI and VestingAccount
+// Interfaces and creates a registry of it's concrete implementations
+func RegisterInterfaces(registry types.InterfaceRegistry) {
+	registry.RegisterInterface(
+		"cosmos_sdk.auth.v1.vesting",
+		(*exported.VestingAccount)(nil),
+		&ContinuousVestingAccount{},
+		&DelayedVestingAccount{},
+		&PeriodicVestingAccount{},
+	)
+	registry.RegisterInterface(
```

Comment:
This should be `RegisterImplementations`. The interface itself was already registered by the main auth module.